tests/tck/features/insert/InsertIfNotExists.feature

```diff
@@ -117,7 +117,7 @@ Feature: Insert vertex and edge with if not exists
       | like | src   | dst     |
       | 100  | "Tom" | "Conan" |
     # insert multi vertex multi tags
-    When executing query:
```

Comment:
What's the error?

Reply:
Insert student for the first time

Reply:
I see in previous stage contains `try to execute query` already. So I don't add it is the resolution.

Reply:
fix

Reply:
And it's same to the other modification.